loleaflet/src/control/Control.NotebookbarBuilder.js

```diff
@@ -415,7 +415,7 @@ L.Control.NotebookbarBuilder = L.Control.JSDialogBuilder.extend({
 			button.id = buttonId;
 			button.setAttribute('alt', id);
 
-			if (data.id === 'FontColor' || data.id === 'BackColor' || data.id === 'BackgroundColor') {
+			if (data.id === 'FontColor' || data.id === 'BackColor' || data.id === 'BackgroundColor' || data.id === 'Color' || data.id === 'CharBackColor' || data.id === 'XLineColor' || data.id === 'FillColor') {
```

Comment:
please remove this condition at all.
We want dropdown in all color pickers, no sense to explicitly add list of them here.